GTBitsOfGood/southface
# Nabeel/115 report notes

## Render notes in report builder

Issue Number(s): #115.

This PR matches the rendering of notes on report builder page to that described in the figma. It also adds an accordion component to view excess of two notes.

### Checklist

- Notes section is added on each card of the report
- Notes are editable and deletable
- render only two notes by default
- onClick of view "X" more notes, all notes are rendered
- collapse all notes is implemented

### How to Test

- Add notes to project report by clicking on a standard, viewing notes, and using the "add notes" button
- May have to refresh page
- Go to report builder
- Observe the newly added notes
- Interact with notes as you would on the standard modal page (edit, delete); may require refresh
- Try adding more than two notes; observe the "view x notes" button and click it to view the remaining notes

I suspect the issues with requiring refresh is a bigger problem with state management and I think this should be left to the bug-fixing work for now
